Userland/reboot.cpp

```diff
@@ -0,0 +1,13 @@
+#include <Kernel/Syscall.h>
+
+
+int main(int argc, char** argv)
+{
+    static_cast<void>(argc);
+    static_cast<void>(argv);
+    syscall(SC_reboot);
```

Comment:
Let's layer this slightly differently. The only user of `syscall` should really be `LibC`.
Instead, let's add a `reboot()` syscall wrapper in `unistd.h` and call that from here.